src/node/proposals.h

```diff
@@ -46,7 +46,8 @@ namespace ccf
    * Or more compact:
    *
    *  local tables, param = ...
-   *  return Calls:call(Puts:put("table", "key", tables["values"]:get(param))
+   *  return Calls:call(Puts:put("table", "key",
+   * tables["ccf.values"]:get(param))
```

Comment:
Extremely minor: Although this is a comment (and we don't have a Lua coding style), it could be manually formatted for readability to:

```
   *  return Calls:call(
   *    Puts:put("table", "key", tables["ccf.values"]:get(param))

```

Reply:
Will fix